src/App.jsx

```diff
@@ -14,45 +16,109 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const rev = 'reverse';
+export const alphabet = 'alphabet';
+export const alphabetRev = 'alphabet reverse';
+export const length = 'length';
+export const lengthRev = 'length reverse';
+
+export const App = () => {
+  const [visibleGoods, setVisibleGoods] = useState(goodsFromServer);
+  const [sortField, setSortField] = useState('');
+
+  const searchClass = (allClass) => {
```

Comment:
```suggestion
  const isReverseSelected = (currentSortField) => {
```

For your approach better to rename to more descriptive names - the function is checking if the sort field has reversed and returns true/false, so better to show this in function name, also it takes sort fields, not classes